GITHUB_API_NUANCE.md

```diff
@@ -0,0 +1,80 @@
+# Nuance in the GitHub API with Probot
+
+Many of the applications in this repository use the [Probot](https://probot.github.io/) framework to power GitHub apps.
+In the source code of some of these apps (e.g., `bundle-size`) you will notice that we define an environment variable
+called `GITHUB_ACCESS_TOKEN`, which should be set to a [personal access token](https://github.com/settings/tokens)
+generated by the user:
+
+###### [bundle-size/.env.example](https://github.com/ampproject/amp-github-apps/blob/33969c884f51a5fcfe342202b1935145b64efd1b/bundle-size/.env.example#L25-L26)
```

Comment:
Nit: This doesn't really need to be a heading, can just be `*[bundle-size/.env.example](...)*`

Reply:
This is a hack for formatting :)